.bowerrc

```diff
@@ -1,3 +1,4 @@
 {
-  "directory": "app/assets/bower_components"
+  "directory": "app/assets/bower_components",
+  "interactive": false
```

Comment:
Or we can add [resolutions](http://stackoverflow.com/a/24281191/394013) to `bower.json`.